<launch>

  <node  name="cloud_node" pkg="rslidar_pointcloud" type="cloud_node" output="screen" >
    <param name="curves_path" value="/data/LM128/curves.csv" />
    <param name="angle_path" value="/data/LM128/angle.csv" />
    <param name="channel_path" value="/data/LM128/ChannelNum.csv" />
  </node>

</launch>
